api_test/CMakeLists.txt

```diff
@@ -34,3 +34,5 @@ endif()
 if (MSVC AND MSVC_VERSION GREATER_EQUAL 1910)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /std:c11")
 endif()
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DCMARK_USE_CMAKE_HEADERS")
```

Comment:
I think that this is better written as:

```suggestion
add_compile_definitions(CMARK_USE_CMAKE_HEADERS)
```

Reply:
You should do that at the top-level CMakeLists.txt only.  It will be applied to any sub-CMakeList.txt (this is why `target_compile_definitions` is generally preferable - it gives you better control).  In general, modifying `CMAKE_C_FLAGS` (or more generally `CMAKE_<LANG>_FLAGS`) is discouraged - that is meant to be controlled by the user only.